packages/snap-preact/components/src/components/Templates/RecommendationBundle/BundleCTA.tsx

```diff
@@ -32,37 +35,42 @@ export const BundledCTA = observer((properties: BundledCTAProps): JSX.Element =>
 
 	const { ctaSlot, cartStore, onAddToCart, ctaIcon, ctaButtonText, ctaButtonSuccessText, treePath } = props;
 
+	const lastPath = treePath?.lastIndexOf(' ');
+	const modifiedTreePath = treePath?.slice(0, lastPath);
```

Comment:
I'm guessing that the `mergeProps()` usage is causing the need for this? Related to a previous comment, what is the purpose of using mergeProps here?